djangocms_snippet/cms_config.py

```diff
@@ -0,0 +1,21 @@
+from cms.app_base import CMSAppConfig
+
+from django.conf import settings
+
+from .models import Snippet
+
+
+class SnippetCMSAppConfig(CMSAppConfig):
+    djangocms_versioning_enabled = getattr(
+        settings, 'DJANGOCMS_SNIPPET_VERSIONING_ENABLED', True)
+
```

Comment:
```suggestion
djangocms_versioning_enabled = getattr(
        settings, 'DJANGOCMS_SNIPPET_VERSIONING_ENABLED', True
)
```

Reply:
Adding this, and changing default to False, as it makes more sense!